src/test/java/org/apache/sling/pipes/internal/ACLPipeTest.java

```diff
@@ -0,0 +1,136 @@
+package org.apache.sling.pipes.internal;
```

Comment:
also @dichaudh you miss a license here, which make rat plugin fails (try `mvn clean install`)

Reply:
Thanks @npeltier  for the feedback , raised a new commit for the changes